mico-core/src/test/java/io/github/ust/mico/core/ApplicationControllerTests.java

```diff
@@ -490,46 +500,59 @@ public void getStatusOfApplication() throws Exception {
         int memoryUsage2 = 70;
         int cpuLoad2 = 40;
 
+        String startTimePod1 = new Date().toString();
+        int restartsPod1 = 0;
+        String startTimePod2 = new Date().toString();
+        int restartsPod2 = 0;
+
         MicoApplicationStatusDTO micoApplicationStatus = new MicoApplicationStatusDTO();
         MicoServiceStatusDTO micoServiceStatus = new MicoServiceStatusDTO();
 
         // Set information for first pod of a MicoService
         KubernetesPodInfoDTO kubernetesPodInfo1 = new KubernetesPodInfoDTO();
         kubernetesPodInfo1
-                .setHostIp(hostIp)
-                .setNodeName(nodeName)
-                .setPhase(podPhase)
-                .setPodName(podName1)
-                .setMetrics(new KuberenetesPodMetricsDTO()
-                        .setAvailable(false)
-                        .setCpuLoad(cpuLoad1)
-                        .setMemoryUsage(memoryUsage1));
+            .setHostIp(hostIp)
+            .setNodeName(nodeName)
+            .setPhase(podPhase)
+            .setPodName(podName1)
+            .setAge(startTimePod1)
+            .setRestarts(restartsPod1)
+            .setMetrics(new KuberenetesPodMetricsDTO()
+                .setAvailable(false)
+                .setCpuLoad(cpuLoad1)
+                .setMemoryUsage(memoryUsage1));
 
         // Set information for second pod of a MicoService
         KubernetesPodInfoDTO kubernetesPodInfo2 = new KubernetesPodInfoDTO();
         kubernetesPodInfo2
-                .setHostIp(hostIp)
-                .setNodeName(nodeName)
-                .setPhase(podPhase)
-                .setPodName(podName2)
-                .setMetrics(new KuberenetesPodMetricsDTO()
-                        .setAvailable(true)
-                        .setCpuLoad(cpuLoad2)
-                        .setMemoryUsage(memoryUsage2));
+            .setHostIp(hostIp)
+            .setNodeName(nodeName)
+            .setPhase(podPhase)
+            .setPodName(podName2)
+            .setAge(startTimePod2)
+            .setRestarts(restartsPod2)
+            .setMetrics(new KuberenetesPodMetricsDTO()
+                .setAvailable(true)
+                .setCpuLoad(cpuLoad2)
+                .setMemoryUsage(memoryUsage2));
 
         // Set deployment information for MicoService
         micoServiceStatus
             .setName(TestConstants.SERVICE_NAME)
             .setShortName(SHORT_NAME)
             .setVersion(VERSION)
             .setAvailableReplicas(availableReplicas)
+            .setUsingApplications(Collections.singletonList(new MicoUsingApplicationDTO(otherMicoApplication.getName(), otherMicoApplication.getShortName(), otherMicoApplication.getVersion())))
```

Comment:
Consider using `CollectionUtils.toList(T... items)`or at least not different functions to create lists.